src/js/node/stream/Readable.hx

```diff
@@ -205,6 +205,8 @@ typedef ReadableNewOptions = {
 	@:optional var highWaterMark:Int;
 	@:optional var encoding:String;
 	@:optional var objectMode:Bool;
+	@:optional var read:Int->Void;
+	@:optional var destroy:Error->(Error->Void)->Void;
```

Comment:
Seems like these are actually `Null<Error>`.

Reply:
Thanks your review comment. I fixed it.